### PR TITLE
Adds a separate page for Features

### PR DESCRIPTION
**Pull request by kiwicopple**

### CODE REVIEW

File: apps/reference/docs/guides/features.mdx [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+| Storage                    | CDN                    | `beta`  |
+| Edge Functions             |                        | `beta`  |
+| Auth                       | OAuth Providers        | `beta`  |
+| Auth                       | Passwordless           | `beta`  |
+| Auth                       | Next.js Auth Helpers   | `alpha` |
+| Auth                       | SvelteKit Auth Helpers | `alpha` |
+| Public API                 |                        | `beta`  |
+| CLI                        |                        | `beta`  |
+| Client Library: JavaScript |                        | `GA`    |
+| Client Library: Dart       |                        | `beta`  |
```
Comment (dannykng) [inline apps/reference/docs/guides/features.mdx:170]:
```suggestion
| Product                    | Feature                | Stage   |
| -------------------------- | ---------------------- | ------- |
| Database                   | Postgres               | `GA`    |
| Database                   | Triggers               | `GA`    |
| Database                   | Functions              | `GA`    |
| Database                   | Extensions             | `GA`    |
| Database                   | Full Text Search       | `GA`    |
| Database                   | Webhooks               | `alpha` |
| Database                   | Point-in-Time Recovery | `alpha` |
| Database                   | Vault                  | `alpha` |
| Studio                     |                        | `GA`    |
| Realtime                   | Postgres CDC           | `beta`  |
| Realtime                   | Broadcast              | `alpha` |
| Realtime                   | Presence               | `alpha` |
| Storage                    |                        | `beta`  |
| Storage                    | CDN                    | `beta`  |
| Edge Functions             |                        | `beta`  |
| Auth                       | OAuth Providers        | `beta`  |
| Auth                       | Passwordless           | `beta`  |
| Auth                       | Next.js Auth Helpers   | `alpha` |
| Auth                       | SvelteKit Auth Helpers | `alpha` |
| Admin API                  |                        | `beta`  |
| CLI                        |                        | `beta`  |
| Client Library: JavaScript |                        | `GA`    |
| Client Library: Dart       |                        | `beta`  |
```

File: apps/reference/docs/guides/features.mdx [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+
+## Auth
+
+### Email & Password Logins
+
+Build email logins for your application or website. [Docs](/docs/guides/auth/auth-email).
+
+### Magic Links
+
+Build passwordless logins for your application or website.[Docs](/guides/auth/auth-magic-link).
```
Comment (dannykng) [inline apps/reference/docs/guides/features.mdx:57]:
```suggestion
Build passwordless logins for your application or website.[Docs](/docs/guides/auth/auth-magic-link).
```

File: apps/reference/docs/guides/features.mdx [hunk 3]
```diff
@@ -0,0 +1,170 @@
...
+
+Cache large files using the Supabase CDN. [Docs](/docs/guides/storage-cdn).
+
+<br />
+
+## Edge Functions
+
+### Deno Edge Functions
+
+Globally distributed Typescript functions to execute custom business logic. [Docs](/docs/guides/functions).
```
Comment (dannykng) [inline apps/reference/docs/guides/features.mdx:126]:
```suggestion
Globally distributed TypeScript functions to execute custom business logic. [Docs](/docs/guides/functions).
```

File: apps/reference/docs/guides/features.mdx [hunk 4]
```diff
@@ -0,0 +1,170 @@
...
+
+Receive your database changes through websockets. [Docs](/docs/guides/realtime/postgres-cdc).
+
+### User Broadcasting
+
+Send messages between connected users through websockets. [Link](/docs/guides/realtime#broadcast).
+
+### User Presence
+
+Synchronize shared state across your users, including online status and typing indicators. [Link](/docs/guides/realtime#presence).
```
Comment (dannykng) [inline apps/reference/docs/guides/features.mdx:101]:
```suggestion
Send messages between connected users through websockets. [Docs](/docs/guides/realtime#broadcast).

### User Presence

Synchronize shared state across your users, including online status and typing indicators. [Docs](/docs/guides/realtime#presence).
```

File: apps/reference/docs/guides/features.mdx [hunk 5]
```diff
@@ -0,0 +1,170 @@
...
+
+Provide phone logins using a 3rd-party SMS provider. [Docs](/docs/guides/auth/auth-twilio).
+
+### Row Level Security
+
+Control the data each user can access with Postgres Policies. [Docs](/docs/guides/auth/row-level-security).
+
+### Serverside Auth Helpers
+
+Helpers for implementing user authentication in popular frameworks like [Next.js](/docs/guides/auth/auth-helpers/nextjs) and [SveltKit](/docs/guides/auth/auth-helpers/sveltekit)
```
Comment (dannykng) [inline apps/reference/docs/guides/features.mdx:73]:
```suggestion
Helpers for implementing user authentication in popular frameworks like [Next.js](/docs/guides/auth/auth-helpers/nextjs) and [SvelteKit](/docs/guides/auth/auth-helpers/sveltekit)
```

File: apps/reference/docs/architecture.mdx [hunk 6]
```diff
@@ -2,38 +2,158 @@
...
+- Official Docs: [postgresql.org/docs](https://www.postgresql.org/docs/current/index.html)
+- Source code: [github.com/postgres/postgres](https://github.com/postgres/postgres) (mirror)
+- License: [PostgreSQL Licence](https://www.postgresql.org/about/licence/)
+- Language: C
+
+### Studio (Dashboard)
+
+An open source Dashboard for managing your database and services.
+
+- Official Docs: [supabase.com/docs](/docs)
```
Comment (dannykng) [inline apps/reference/docs/architecture.mdx:36]:
```suggestion
- Official Docs: [Supabase docs](/docs)
```

File: apps/reference/docs/architecture.mdx [hunk 7]
```diff
@@ -2,38 +2,158 @@
...
+- Source code: [github.com/supabase/supabase](https://github.com/supabase/supabase/tree/master/studio)
+- License: [Apache 2](https://github.com/supabase/supabase/blob/master/LICENSE)
+- Language: TypeScript
+
+### GoTrue (Auth)
+
+A JWT-based API for managing users and issuing access tokens. This integrates with PostgreSQL's Row Level Security and the API servers.
+
+- Official Docs: [supabase.com/docs/reference/auth](/docs/reference/auth)
```
Comment (dannykng) [inline apps/reference/docs/architecture.mdx:45]:
```suggestion
- Official Docs: [Supabase Auth reference docs](/docs/reference/auth)
```

File: apps/reference/docs/architecture.mdx [hunk 8]
```diff
@@ -2,38 +2,158 @@
...
+- Official Docs: [postgrest.org](https://postgrest.org/)
+- Source code: [github.com/PostgREST/postgrest](https://github.com/PostgREST/postgrest)
+- License: [MIT](https://github.com/PostgREST/postgrest/blob/main/LICENSE)
+- Language: Haskel
+
+### Realtime (API & multiplayer)
+
+A scalable websocket engine for managing user Presence, broadcasting messages, and streaming database changes.
+
+- Official Docs: [supabase.com/docs/guides/realtime](/docs/guides/realtime)
```
Comment (dannykng) [inline apps/reference/docs/architecture.mdx:64]:
```suggestion
- Official Docs: [Supabase Realtime docs](/docs/guides/realtime)
```

File: apps/reference/docs/architecture.mdx [hunk 9]
```diff
@@ -2,38 +2,158 @@
...
+- Source code: [github.com/supabase/realtime](https://github.com/supabase/realtime)
+- License: [Apache 2](https://github.com/supabase/realtime/blob/master/LICENSE)
+- Language: Elixir
+
+### Storage API (large file storage)
+
+An S3-compatible object storage service that stores metadata in Postgres.
+
+- Official Docs: [supabase.com/docs/reference/storage](/docs/reference/storage)
```
Comment (dannykng) [inline apps/reference/docs/architecture.mdx:73]:
```suggestion
- Official Docs: [Supabase Storage reference docs](/docs/reference/storage)
```

File: apps/reference/docs/architecture.mdx [hunk 10]
```diff
@@ -2,38 +2,158 @@
...
+- Official Docs: [deno.land](https://deno.land/)
+- Source code: [github.com/denoland/deno](https://github.com/denoland/deno)
+- License: [MIT](https://github.com/denoland/deno/blob/main/LICENSE.md)
+- Language: TypeScript / Rust
+
+### postgres-meta (Database management)
+
+A RESTful API for managing your Postgres. Fetch tables, add roles, and run queries.
+
+- Official Docs: [supabase.github.io/postgres-meta](https://supabase.github.io/postgres-meta/)
```
Comment (dannykng) [inline apps/reference/docs/architecture.mdx:91]: This link doesn't seem to provide much right now

Comment (kiwicopple) [reply apps/reference/docs/architecture.mdx:91]: yeah, I will get to these docs shortly (I'll add them to the reference docs

File: apps/reference/docs/architecture.mdx [hunk 11]
```diff
@@ -2,38 +2,156 @@
...
-| Auth                       | Passwordless           | Beta  | [Link](/guides/auth/auth-magic-link)             |
-| Auth                       | Next.js Auth Helpers   | Alpha | [Link](/docs/guides/auth/auth-helpers/nextjs)    |
-| Auth                       | SvelteKit Auth Helpers | Alpha | [Link](/docs/guides/auth/auth-helpers/sveltekit) |
-| Public API                 |                        | Beta  | [Link](/docs/reference/api)                      |
-| CLI                        |                        | Beta  | [Link](/docs/guides/cli)                         |
-| Client Library: JavaScript |                        | GA    | [Link](/docs/reference/javascript)               |
-| Client Library: Dart       |                        | Beta  | [Link](/docs/reference/dart)                     |
+### PostgreSQL (Database)
+
+PostgreSQL is the core of Supabase. We do not abstract the PostgreSQL database—you can access it and use it with full privileges. We simply provide tools which makes PostgreSQL as easy to use as Firebase.
```
Comment (awalias) [inline apps/reference/docs/architecture.mdx:23]: just a small thing "database-you" looks like one word

File: apps/reference/docs/architecture.mdx [hunk 12]
```diff
@@ -2,38 +2,156 @@
...
+
+We're deliberate about adding a new tool, and prefer instead to extend an existing one.
+This is the opposite of many cloud providers whose product offering expands into niche use-cases. We provide _primitives_ for developers, which allow them to achieve any goal.
+Less, but better.
+
+### Everything is portable
+
+To avoid lock-in, we make it easy to migrate in and out. Our cloud offering is compatible with our self-hosted product.
+We use existing standards to increase portability (like pg_dump an CSV files). If a new standard emerges which competes with a "Supabase" approach, we will deprecate the approach in favor of the standard.
+This forces us compete on experience. If we are not the best Postgres hosting service, then developers deserve to migrate away.
```
Comment (awalias) [inline apps/reference/docs/architecture.mdx:139]: 'deserve' doesn't quite fit, maybe just:

"This forces us compete on experience. We aim to be the best Postgres hosting service"

File: apps/reference/docs/architecture.mdx [hunk 13]
```diff
@@ -2,38 +2,156 @@
...
+
+### Play the long game
+
+We sacrifice short-term wins for long-term gains. For example, it is tempting to run a fork of Postgres with additional functionality which only our customers need.
+Instead, we prefer to support efforts to upstream missing functionality so that the entire community benefits. This has the additional benefit of ensuring portability.
```
Comment (awalias) [inline apps/reference/docs/architecture.mdx:144]: portability _and longevity_?

File: apps/reference/docs/architecture.mdx [hunk 14]
```diff
@@ -2,38 +2,156 @@
...
+"Developers" are a specific profile of user: they are _builders_.
+When assessing impact as a function of effort, developers have a large efficiency due to the type of products and systems they can build.
+As the profile of a developer changes over time, Supabase will continue to evolve the product to fit this evolving profile.
+From another perspective, we will not develop a product beyond this profile: we don't aim to become a tool for everyone - only developers.
+
+### Support existing tools
+
+Supabase supports existing tools and communities wherever possible. Supabase is more like a "community of communities" - each tool typically has its own community
+which we work with.
+Open source is something we approach collaboratively: we employ maintainers, sponsor projects, invest in businesses, and develop our own open source tools.
```
Comment (awalias) [inline apps/reference/docs/architecture.mdx:157]: worth linking to your blog post?

**Review by awalias — APPROVED** [review]
💯

File: apps/reference/docs/architecture.mdx (hunk 11 above)
Comment (dannykng) [reply apps/reference/docs/architecture.mdx]: Ah, this is just a GitHub display thing. That's actually an em dash.

**Pull request merged**